<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cCharacterWindow</class>
 <widget class="QWidget" name="cCharacterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[character] - </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_lpTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_lpBasic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0,0">
         <item row="9" column="1" colspan="5">
          <widget class="cTextEdit" name="m_lpDescription"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Date of Death:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="cDoubleSpinBox" name="m_lpHeight">
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="cLineEdit" name="m_lpCreature"/>
         </item>
         <item row="3" column="3">
          <widget class="cLineEdit" name="m_lpPlaceOfBirth"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Place of Death:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Creature:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Place of Birth:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="cDateEdit" name="m_lpDayOfDeath">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="cDateEdit" name="m_lpDayOfBirth">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Date of Birth:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="cLineEdit" name="m_lpPlaceOfDeath"/>
         </item>
         <item row="1" column="1">
          <widget class="cCheckBox" name="m_lpMainCharacter">
           <property name="text">
            <string>Main Character</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Weight:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="cDoubleSpinBox" name="m_lpWeight">
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Middle Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>First Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="cLineEdit" name="m_lpFirstName"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Description:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="cLineEdit" name="m_lpMiddleName"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Last Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="cLineEdit" name="m_lpLastName"/>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Hair Color:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Hair Cut:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Hair Length:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="cLineEdit" name="m_lpHairColor"/>
         </item>
         <item row="6" column="5">
          <widget class="cLineEdit" name="m_lpHairCut"/>
         </item>
         <item row="7" column="5">
          <widget class="cLineEdit" name="m_lpHairLength"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Figure:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="cLineEdit" name="m_lpFigure"/>
         </item>
         <item row="8" column="4">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Job:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Skin:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="cLineEdit" name="m_lpSkin"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>School:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Spoken Languages:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Nature:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="cLineEdit" name="m_lpNature"/>
         </item>
         <item row="8" column="1">
          <widget class="cTextEdit" name="m_lpSchool"/>
         </item>
         <item row="8" column="3">
          <widget class="cTextEdit" name="m_lpSpokenLanguages"/>
         </item>
         <item row="8" column="5">
          <widget class="cTextEdit" name="m_lpJob"/>
         </item>
         <item row="1" column="5" rowspan="4">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Gender</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="cRadioButton" name="m_lpMale">
              <property name="text">
               <string>Male</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="cRadioButton" name="m_lpFemale">
              <property name="text">
               <string>Female</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="cRadioButton" name="m_lpOther">
              <property name="text">
               <string>Other</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="cLineEdit" name="m_lpTitle"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Title:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Nick Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="cLineEdit" name="m_lpNickName"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_lpImages">
      <attribute name="title">
       <string>Images</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QScrollArea" name="m_lpScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>681</width>
            <height>464</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="m_lpLayout"/>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ctextedit.h</header>
  </customwidget>
  <customwidget>
   <class>cLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
  <customwidget>
   <class>cCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ccheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>cRadioButton</class>
   <extends>QRadioButton</extends>
   <header>cradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>cDateEdit</class>
   <extends>QDateEdit</extends>
   <header>cdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>cDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>cdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lpTab</tabstop>
  <tabstop>m_lpFirstName</tabstop>
  <tabstop>m_lpMiddleName</tabstop>
  <tabstop>m_lpLastName</tabstop>
  <tabstop>m_lpMainCharacter</tabstop>
  <tabstop>m_lpCreature</tabstop>
  <tabstop>m_lpNickName</tabstop>
  <tabstop>m_lpTitle</tabstop>
  <tabstop>m_lpMale</tabstop>
  <tabstop>m_lpFemale</tabstop>
  <tabstop>m_lpOther</tabstop>
  <tabstop>m_lpDayOfBirth</tabstop>
  <tabstop>m_lpPlaceOfBirth</tabstop>
  <tabstop>m_lpDayOfDeath</tabstop>
  <tabstop>m_lpPlaceOfDeath</tabstop>
  <tabstop>m_lpHeight</tabstop>
  <tabstop>m_lpFigure</tabstop>
  <tabstop>m_lpWeight</tabstop>
  <tabstop>m_lpSkin</tabstop>
  <tabstop>m_lpNature</tabstop>
  <tabstop>m_lpHairColor</tabstop>
  <tabstop>m_lpHairCut</tabstop>
  <tabstop>m_lpHairLength</tabstop>
  <tabstop>m_lpSchool</tabstop>
  <tabstop>m_lpSpokenLanguages</tabstop>
  <tabstop>m_lpJob</tabstop>
  <tabstop>m_lpDescription</tabstop>
  <tabstop>m_lpScrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
